custom_components/airthings_wave/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "domain": "airthings_wave",
   "name": "Airthings Wave",
-  "version": "3.1.0",
+  "version": "3.2.0",
   "documentation": "https://github.com/custom-components/sensor.airthings_wave/",
   "issue_tracker": "https://github.com/custom-components/sensor.airthings_wave/issues",
   "dependencies": [],
   "codeowners": ["@MartyTremblay","@sverrham"],
   "requirements": [
-    "bluepy==1.3.0"
+    "bleak"
```

Comment:
Pin to specific version of bleak to avoid surprise updates?

Reply:
Pinned to 0.14.3, thanks for the feedback